input/pagecontent/StructureDefinition-au-erequesting-servicerequest-imag-intro.md

```diff
@@ -1,11 +1,26 @@
 {% include rrv-dev-note.md %}
 
 ### Profile specific implementation guidance
-- `ServiceRequest.code`:
-  - **SHOULD** be selected from the preferred <a href="ValueSet-ranzcr-radiology-referral.html">RANZCR Radiology Referral</a> value set
-  - if a system is unable to provide a code from the preferred <a href="ValueSet-ranzcr-radiology-referral.html">RANZCR Radiology Referral</a> value set, it is preferred to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set over only supplying local codes
-  - coding is preferred over supplying only free text with no codes
-- The use of coding can vary significantly across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.
-- See guidance on the construction of identifiers in each Identifier profile and the section on [Business Identifiers](https://build.fhir.org/ig/hl7au/au-fhir-base/generalguidance.html#business-identifiers) in AU Base.
+
+#### `ServiceRequest.code`
+The use of coding for the requested exam can vary significantly across systems, filler systems need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Placer systems **SHALL** populate `ServiceRequest.code.text` so that fillers can at least display the service requested, even if they do not recognise the code supplied.
+
+Placer systems **SHALL** adhere to the following order of preference when populating the `ServiceRequest.code`:
+
+1. A code selected from the [RANZCR Radiology Referral](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/ValueSet-ranzcr-radiology-referral.html) Value Set is the preferred option
+2. A code selected from the SNOMED CT [Imaging Procedure](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/imaging-procedure-1) Value Set is the second most preferred option
+3. A local code is preferred over free text, preferably from a published and accessible FHIR code system​
+4. Free text is the least preferred option which should not be used if a local code is known
```

Comment:
@michael-wilson-au, "which should not be used if a local code is known" introduced a conflict as ServiceRequest.code.text is 1..1.

Reply:
Addressed by Michael below - no change required.